src/actions/users.js

```diff
@@ -1122,6 +1122,153 @@ export function switchLdapToEmail(ldapPassword, email, emailPassword, mfaCode =
     );
 }
 
+export function createUserAccessToken(userId, description) {
+    return async (dispatch, getState) => {
+        dispatch({type: UserTypes.CREATE_USER_ACCESS_TOKEN_REQUEST});
+
+        let data;
+        try {
+            data = await Client4.createUserAccessToken(userId, description);
+        } catch (error) {
+            dispatch({type: UserTypes.CREATE_USER_ACCESS_TOKEN_FAILURE, error});
+            return {error};
```

Comment:
Don't we want to do that on all actions? If we get a 401 back we want to log out the user right?